src/modules/site-v2/base/views/tools/pairwise_indel_finder.py

```
@@ -58,6 +58,12 @@ def results_columns():
     },
   ]
 
+def try_get_sv_strains(species):
+  try:
+    return get_sv_strains(species)
+  except:
+    return []
```

@vlagrassa -- we prefer avoiding silent errors. If appropriate, consider logging here that the species was not found. We can always use that data-point in the logs to troubleshoot issues.

---

Fixed in 6c8c326d22b50ff43e749d8a3519d88249b796ae